source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -206,7 +206,7 @@ You can write an Extended JSON string from an instance of ``Document`` or
 ``BsonDocument`` by calling the ``toJson()`` method, optionally passing it an
 instance of ``JsonWriterSettings`` to specify the Extended JSON format.
 
-In this example, we output the Extended JSON in the "Relaxed" format.
+In this example, we output the Extended JSON in the Relaxed mode format.
```

Comment:
It's not that clear what the correct term is since both "Relaxed Mode" and "relaxed format" are used in the same bullet item on https://docs.mongodb.com/manual/reference/mongodb-extended-json/ but whichever one you determine is the better one to use, please make sure to stay consistent with the capitalization.

Reply:
Thanks for the feedback. Based off the [Sever Docs Extended JSON Section](https://docs.mongodb.com/manual/reference/mongodb-extended-json/), it seems like the correct capitalization is "Relaxed mode" (capital "R" lowercase "m" when not used as a title). I changed the other instance of a non-title instance of "Relaxed mode" to match this formatting in https://github.com/mongodb/docs-java/pull/57/commits/9c3c131ad570f67731a635a8969b6636907f3e8a. Let me know your thoughts on this solution.